src/java/src/main/java/ai/onnxruntime/genai/NamedTensors.java

```diff
@@ -3,9 +3,18 @@
  */
 package ai.onnxruntime.genai;
 
+/**
+ * This class is an intermediate storage class that bridges the output of preprocessing and the
```

Comment:
This is just my understanding.

Reply:
Yeah, the named tensors are just a way to have a list of tensors with names that match up with model input names. Perhaps:

"NamedTensors is a list of tensors with names that match up with model input names."